nbgitpuller/templates/page.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+    <meta charset="utf-8">
+
+    <title>{% block title %}Jupyter Notebook{% endblock %}</title>
+    {% block favicon %}<link id="favicon" rel="shortcut icon" type="image/x-icon" href="{{static_url("base/images/favicon.ico") }}">{% endblock %}
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="{{static_url("components/jquery-ui/themes/smoothness/jquery-ui.min.css") }}" type="text/css" />
+    <link rel="stylesheet" href="{{static_url("components/jquery-typeahead/dist/jquery.typeahead.min.css") }}" type="text/css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    {% block stylesheet %}
+    <link rel="stylesheet" href="{{ static_url("style/style.min.css") }}" type="text/css"/>
+    {% endblock %}
+    <link rel="stylesheet" href="{{ base_url }}custom/custom.css" type="text/css" />
+    <script src="{{static_url("components/es6-promise/promise.min.js")}}" type="text/javascript" charset="utf-8"></script>
+    <script src="{{static_url('components/react/react.production.min.js')}}" type="text/javascript"></script>
+    <script src="{{static_url('components/react/react-dom.production.min.js')}}" type="text/javascript"></script>
+    <script src="{{static_url('components/create-react-class/index.js')}}" type="text/javascript"></script>
+    <script src="{{static_url('components/requirejs/require.js') }}" type="text/javascript" charset="utf-8"></script>
+    <script>
+      require.config({
+          {% if version_hash %}
+          urlArgs: "v={{version_hash}}",
+          {% endif %}
+          baseUrl: '{{static_url("", include_version=False)}}',
+          paths: {
+            'auth/js/main': 'auth/js/main.min',
```

Comment:
Let's remove the ones we don't need? I think we can get rid of a lot of these!

Reply:
I cheated and copied the `page.html` file from [jupyter_server](https://github.com/jupyter-server/jupyter_server/blob/master/jupyter_server/templates/page.html) which should have only the things we need. I'm not sure if I'm right though :grimacing: